src/EditorFeatures/CSharpTest/GenerateConstructor/GenerateConstructorTests.cs

```diff
@@ -2,13 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
```

Comment:
Won't let me comment on the line specifically, but line 28 needs a NRT on the DiagnosticAnalyzer